easybuild/tools/toolchain/compiler.py

```diff
@@ -231,9 +231,17 @@ def _set_compiler_flags(self):
         fflags = [self.options.option(x) for x in self.COMPILER_F_FLAGS + self.COMPILER_F_UNIQUE_FLAGS \
                   if self.options.get(x, False)]
 
+        # Allow a user-defined default optimisation
+        default_opt_level = build_option('default_opt_level')
+        if default_opt_level in self.COMPILER_OPT_FLAGS:
+            default_opt = default_opt_level
+        else:
+            raise EasyBuildError("Unknown value for default optimisation: %s (possibilities are %s)" %
+                                 (default_opt_level, self.COMPILER_OPT_FLAGS))
+
```

Comment:
Hmm, leave it there, it's pretty cheap, and we may change things later such that `default_opt_level` comes from somewhere else